src/main/java/com/minecolonies/sounds/FishermanSounds.java

```diff
@@ -0,0 +1,83 @@
+package com.minecolonies.sounds;
+
+import com.minecolonies.util.SoundUtils;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.Random;
+
+/**
+ * Class containing the fisherman sounds.
+ */
+public class FishermanSounds
+{
+    /**
+     * Random generator.
+     */
+    private static Random rand = new Random();
+
+    /**
+     * Number of different sounds in this class.
+     */
+    private static final int NUMBER_OF_SOUNDS = 2;
+
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 25;
+
+    /**
+     * Chance to play a basic sound.
+     */
+    private static final int BASIC_SOUND_CHANCE = 100;
+
+    /**
+     * Containing the female fisherman sounds.
+     */
+    public static class Female
+    {
+        public static SoundEvent generalPhrases;
+        public static SoundEvent noises;
+
+        public static SoundEvent iGotOne;
+        public static SoundEvent needFishingRod;
+        public static SoundEvent offToBed;
+    }
+
+    /**
+     * Containing the male fisherman sounds.
+     */
+    public static class Male
+    {
+
+    }
+
+    /**
+     * Plays fisherman sounds.
+     * @param worldIn the world to play the sound in.
+     * @param position the position to play the sound at.
+     * @param isFemale the gender.
+     */
+    public static void playFishermanSound(World worldIn, BlockPos position, boolean isFemale)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Provide multiple methods instead of using "isFemale" to determine which action to take. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2301)